README.md

```diff
@@ -11,43 +11,189 @@ description: Creates an Azure AI Service and deploys the specified models.
+
+1. Open the template (this may take several minutes)
+    [![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](placeholder)
+2. Open a terminal window
+3. Sign into your Azure account:
+
+    ```shell
+     azd auth login --use-device-code
+    ```
+
+4. [any other steps needed for your template]
+5. Provision the Azure resources and deploy your code:
+
+    ```shell
+    azd up
+    ```
+
+6. (Add steps to start up the sample app)
+
+### VS Code Dev Containers
+
+A related option is VS Code Dev Containers, which will open the project in your local VS Code using the [Dev Containers extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers):
+
+1. Start Docker Desktop (install it if not already installed)
+2. Open the project:
+    [![Open in Dev Containers](https://img.shields.io/static/v1?style=for-the-badge&label=Dev%20Containers&message=Open&color=blue&logo=visualstudiocode)](placeholder)
+3. In the VS Code window that opens, once the project files show up (this may take several minutes), open a terminal window.
+4. Sign into your Azure account:
+
+    ```shell
+     azd auth login
+    ```
+
+5. [any other steps needed for your template]
+6. Provision the Azure resources and deploy your code:
+
+    ```shell
+    azd up
+    ```
+
+7. (Add steps to start up the sample app)
+
+8. Configure a CI/CD pipeline:
+
+    ```shell
+    azd pipeline config
+    ```
+
+### Local Environment
+
+#### Prerequisites
+
+(ideally very short, if any)
+
+* Install [azd](https://aka.ms/install-azd)
+  * Windows: `winget install microsoft.azd`
+  * Linux: `curl -fsSL https://aka.ms/install-azd.sh | bash`
+  * MacOS: `brew tap azure/azd && brew install azd`
+* OS
+* Library version
+* This template uses [MODEL 1] and [MODEL 2] which may not be available in all Azure regions. Check for [up-to-date region availability](https://learn.microsoft.com/azure/ai-services/openai/concepts/models#standard-deployment-model-availability) and select a region during deployment accordingly
+  * We recommend using [SUGGESTED REGION]
+* ...
+
+#### Installation
+
+(ideally very short)
+
+* list of any prerequisites
+* ...
+
+#### Quickstart
+
+(Add steps to get up and running quickly)
+
+1. Bring down the template code:
+
+    ```shell
+    azd init --template [name-of-repo]
+    ```
+
+    This will perform a git clone
+
+2. Sign into your Azure account:
+
+    ```shell
+     azd auth login
+    ```
+
+3. [Packages or anything else that needs to be installed]
+
+    ```shell
+    npm install ...
+    ```
+
+4. ...
+5. Provision and deploy the project to Azure:
+
+    ```shell
+    azd up
+    ```
+
+6. (Add steps to start up the sample app)
+
+7. Configure a CI/CD pipeline:
+
+    ```shell
+    azd pipeline config
+    ```
+
+#### Local Development
+
+Describe how to run and develop the app locally
+
+## Guidance
```

Comment:
how about list the 2 useful scripts Zhijie added here
- list available quota of  user's subs
- list api version

Reply:
You have a good point. This will require revision in the future though.